source/process/release-process.md

```diff
@@ -224,6 +224,7 @@ The final release is cut. If an urgent and important issue needs to be addressed
     - Review any changes made to install guides, and test if necessary
     - Run loadtests against the final release build to confirm there are no performance issues
     - Review [Security Policies](https://docs.mattermost.com/process/security.html) page is updated
+    - Update dependancies after release branch is cut
```

Comment:
Actually, at T-0 looks like there's an item

```
Check if any libraries need to be updated in all repositories, including `mattermost-server`, `mattermost-webapp`, `desktop`, `mattermost-mobile` and `mattermost-redux`
```

but I think moving it to T-2 makes sense, and calling it dependancies. @amyblais Maybe we can just tweak to say

```
Update dependancies after release branch is cut in `mattermost-server`, `mattermost-webapp`, `desktop`, `mattermost-mobile` and `mattermost-redux`
```